src/App.js

```diff
@@ -20,37 +20,56 @@ What should a cart item look like? 🤔
 console.log(initialStoreItems)
 
 export default function App() {
-  // const [items, setItems] = useState(initialStoreItems)
+ 
   const [cart, setCart] = useState([])
   // Setup state here...
 
   const addToCart = selection => {
+    // if selected item is not in the cart, the cart will 
+    // from quantity 1
     if (!cart.includes(selection)) {
+      console.log("cart", !cart, "selection", selection)
       selection.quantity = 1
+      //  upgrade state with an array whatever is inside the cart
       setCart([...cart, selection])
     } else {
+      // if selected item includes in the cart, will add more 
+      // selected item on top
       selection.quantity++
+      // update cart whatever inside it.
       setCart([...cart])
       console.log('add to cart', selection)
     }
   }
   const decreaseItem = selection => {
```

Comment:
decreasing and removing item well presented in 2 different functions . i liked it. easy to understand .